bangc-ops/mlu_op.h

```diff
@@ -11474,6 +11474,84 @@ mluOpRoiAlignBackward_v2(mluOpHandle_t handle,
                          const mluOpTensorDescriptor_t grads_image_desc,
                          void *grads_image);
 
+// Group:MsDeformAttnForward
+/*!
+ * @brief Implements a multi-scale deformable attention module used in Deformable-Detr.
+ * For detailed information about Deformable-Detr, see "Deformable DETR: Deformable
+ * Transformers for End-to-End Object Detection."
+ *
+ * @param[in] handle
+ * Handle to an MLUOP context that is used to manage MLU devices and queues
+ * in ::mluOpMsDeformAttnForward operation. For detailed information, see ::mluOpHandle_t.
+ * @param[in] data_value_desc
+ * The descriptor of data_value. For detailed information, see ::mluOpTensorDescriptor_t.
```

Comment:
The descriptor of the tensor \b data_value .

Reply:
done